### PR TITLE
Update Android SFA docs

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

File: docs/migration-guides/sfa-android-v0.1.0-to-v0.3.0.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#### Before (v0.1.0)
+
+```kotlin
+val singleFactorAuthArgs = SingleFactorAuthArgs(TorusNetwork.TESTNET)
+```
+
+#### After (v0.3.0)
+
+```kotlin
+val singleFactorAuthArgs = SingleFactorAuthArgs(Web3AuthNetwork.MAINNET, "YOUR_WEB3AUTH_CLIENT_ID")
```
Comment (shahbaz17) [inline docs/migration-guides/sfa-android-v0.1.0-to-v0.3.0.mdx:30]: Why does SFA android take things directly and Sfa flutter takes it as `SFAParams`?

File: docs/sdk/core-kit/sfa-android/usage.mdx [hunk 2]
```diff
@@ -142,7 +127,7 @@ class MainActivity : AppCompatActivity() {
   private fun signIn() {
     val jwt = JWT(idToken) // idToken is the JWT token obtained from auth provider or your custom JWT provider
     val sub = jwt.getClaim("sub").asString() // get sub claims
-    loginParams = LoginParams("web3auth-firebase-examples", "$sub", "$idToken")
+    loginParams = LoginParams("w3a-firebase-demo", "$sub", "$idToken")
     try {
         torusKey = singleFactorAuth.getKey(loginParams).get()
```
Comment (shahbaz17) [inline docs/sdk/core-kit/sfa-android/usage.mdx:132]: Here, torusKey is the privateKey, so it's better to be direct for a developer who doesn't know what is a torusKey. torusKey is an internal name.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

File: docs/migration-guides/sfa-android-v0.1.0-to-v0.3.0.mdx (hunk 1 above)
Comment (AyushBherwani1998) [reply docs/migration-guides/sfa-android-v0.1.0-to-v0.3.0.mdx:30]: It's same only, there's difference in naming convention, in Android the object is called SingleFactoAuthArgs, and on Flutter it's called the SFAParams.

**Review by shahbaz17 — APPROVED** [review]
lgtm

**Pull request merged**